.dockerignore

```diff
@@ -1,6 +1,8 @@
 test
 app/**/*.test.js
 app/**/*.tests.js
```

Comment:
should probably just be **/*.test.js and **/*.tests.js

Reply:
For sure! :D Addressed in [93b0074]